server/update-workflow-interface/update-workflow-interface.md

```diff
@@ -0,0 +1,217 @@
+# Proposal: Update Workflow Server Interface
```

Comment:
Feel free to ignore, but I don't like the name. Signal is a good name. "Update" is not. This term is so overloaded. I think we should choose a more descriptive name for this abstraction.

Reply:
Prompt? Troll?

I'm thinking about it but haven't come up with anything that's obviously better than update yet.